project/Dependencies.scala

```diff
@@ -5,49 +5,49 @@ object Dependencies {
   object V {
     val bouncyCastle = "1.70"
     val betterFiles = "3.9.1"
-    val cats = "2.8.0"
-    val catsEffect = "3.3.14"
-    val catsRetry = "3.0.0"
-    val circe = "0.14.2"
-    val ciris = "2.3.3"
-    val comcast = "3.1.3"
-    val decline = "2.3.0"
+    val cats = "2.9.0"
+    val catsEffect = "3.4.2"
+    val catsRetry = "3.1.0"
+    val circe = "0.14.3"
+    val ciris = "3.0.0"
```

Comment:
Are we sure we are compatible after this update?

Reply:
I think so @TheMMaciek, according to the changelog. also as far as I checked, configuration from cli and env is loaded properly.

Reply:
They didn't change a lot according to the changelog.

>Add support for Scala.js and Scala Native (https://github.com/vlovgr/ciris/pull/607) @vlovgr
Change to replace gitter badge with discord (https://github.com/vlovgr/ciris/pull/608) @vlovgr
Update lastname in copyright (https://github.com/vlovgr/ciris/pull/609) @vlovgr
Update circe-yaml to 0.14.2 (https://github.com/vlovgr/ciris/pull/618) @vlovgr
Update scala to 2.13.10 (https://github.com/vlovgr/ciris/pull/611) @scala-steward
Update sbt to 1.7.3 (https://github.com/vlovgr/ciris/pull/617) @vlovgr
Update sbt-ci-release to 1.5.11 (https://github.com/vlovgr/ciris/pull/615) @scala-steward
Update sbt-header to 5.8.0 (https://github.com/vlovgr/ciris/pull/616) @scala-steward
Update sbt-mdoc to 2.3.6 (https://github.com/vlovgr/ciris/pull/613) @scala-steward
Update scalafmt-core to 3.6.1 (https://github.com/vlovgr/ciris/pull/619) @scala-steward